### DISCUSSION TITLE
请问可以把bot收到的图片保存到本地吗

### DISCUSSION THREAD

**Discussion by ZenitH-zyh** [GraiaProject/Application#108] [Q&A]
现在我想要把群友发的特定的图存在bot的主机上，请问怎么操作

**Comment by RainyNian** [comment]
Image类里有url

**Comment by I-love-study** [comment]
直接获得Bytes的话就用`await Image.http_to_bytes()`(Image指的是MessageChain中的Image)
如果需要获取单条消息链中的所有Bytes的话，可以使用列表表达式
`images_bytes = [i.http_to_bytes() for i in message.get(Image)]`
至于保存到本地的话，用`open`函数即可
